feature-libs/asm/styles/components/_asm-main-ui.component.scss

```diff
@@ -77,14 +77,16 @@
 
   // Top Bar
   .asm-bar {
-    color: #fff;
-    background-color: #354a5f;
+    background-color: #ffffff;
```

Comment:
colors should be variables. This should be 
`background-color: var(--cx-color-inverse);`